pack/apk.mk

```diff
@@ -45,12 +45,12 @@ package: $(BUILDDIR)/$(TARBALL) \
 	@echo "-------------------------------------------------------------------"
 	sudo apk update
 	mkdir ~/.abuild/
-	echo "PACKAGER_PRIVKEY=/home/${USER}/.abuild/private.rsa" > ~/.abuild/abuild.conf
+	echo "PACKAGER_PRIVKEY=~/.abuild/private.rsa" > ~/.abuild/abuild.conf
 	echo "$(ABUILD_KEY)" | base64 -d > ~/.abuild/private.rsa
```

Comment:
Strictly speaking, it is not about this patch, but while we're here, it seems critical to disable echoing before this output (and enable afterwards).

![image](https://user-images.githubusercontent.com/92749/208265397-880de83d-0c94-44ec-b137-0ecd33a7e436.png)

(The key on the screenshot is generated for testing.)

Reply:
I disable output for this echo.